frontend/src/utils/logger.ts

```diff
@@ -0,0 +1,38 @@
+import { logException, logMessage } from '../sentry.config'
+import { APPLICATION_ENV } from '../utils/credentials'
+
+/**
+ * Logger utility for handling logs and errors.
+ * In development, logs are printed to the console.
+ * In production, logs and errors are sent to Sentry.
+ */
+export const logger = {
+  log: (...args: unknown[]) => {
+    const message = args
+      .map((arg) => (typeof arg === 'string' ? arg : JSON.stringify(arg)))
+      .join(' ')
+
+    if (APPLICATION_ENV === 'development') {
+      console.log(...args)
+    } else {
+      logMessage(message)
+    }
+  },
```

Comment:
I like the `logger` abstraction. Please note that we don't want to treat regular log entries as errors in prod -- don't send them to Sentry.

Reply:
okay modified logger.ts